manifests/repo.pp

```diff
@@ -159,6 +163,13 @@
         Apt::Source['zabbix'] -> Package<|tag == 'zabbix'|>
         Class['Apt::Update']  -> Package<|tag == 'zabbix'|>
       }
+      'windows' : {
+        if ($manage_choco == true) {
+          unless defined(Class['chocolatey']) {
+            include chocolatey
```

Comment:
at the moment I'm a bit unsure if we should keep this here of if chocolatey should be included in a profile

Reply:
Ok, I can remove this call and remove the related entries from fixture.yml and metadata.json.